takhojin/utils/checkToken.js

```diff
@@ -0,0 +1,18 @@
+const secreteKey = process.env.secretKey;
+const jwt = require("jsonwebtoken");
+
+const checkToken = async (req, res, next) => {
+  const token = req.headers.authorization;
+  try {
+    jwt.verify(token, secreteKey);
+    next();
```

Comment:
1. token의 존재 여부를 확인하는 로직을 추가해 주세요.
2. verify 후 payload에 있는 user 정보를 req에 담는 로직을 추가해 주세요.

```suggestion
  try {
    const token = req.headers.authorization;
    // token의 존재 여부를 확인하는 로직 추가
    jwt.verify(token, secreteKey);
    //verify 후 payload에 있는 user 정보를 req에 담는 로직 추가
    
    next();
```